src/pages/en/guides/migrate-to-astro/from-nuxtjs.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Migrating from NuxtJS
+description: Tips for migrating an existing NuxtJS project to Astro
+layout: ~/layouts/MigrationLayout.astro
+stub: true
+framework: NuxtJS
+---
+
+[NuxtJS](https://nuxtjs.org) is a framework for building web applications on top of Vue.
+
+## Key Similarities between Nuxt and Astro
+
+Nuxt and Astro share some similarities that will help you migrate your project:
+
+- Both Nuxt and Astro are modern Javascript static-site generators with similar project file structures. Both use a `src/` folder for your project files and a special `src/pages/` folder for file-based routing. Creating and managing pages for your site should feel familiar.
+
+- Astro has [an official integration for using Vue components](/en/guides/integrations-guide/vue/) and support for NPM packages and community integrations, including several for Vue. You will be able to write Vue UI components for interactivity, and may be able to keep some or all of your existing Nuxt Vue components and dependencies.
+
+- Astro and Nuxt both allow you to use a Headless CMS, APIs or Markdown files for data. You can continue to use your preferred authoring system, and will be able to keep your existing content.
+
+## Key Differences between Nuxt and Astro
+
+When you rebuild your Nuxt site in Astro, you will notice some important differences:
+
+- [MPA vs SPA](/en/concepts/mpa-vs-spa/): Nuxt is a React-based SPA (single-page application). Astro sites are multi-page apps built using `.astro` components, but can also support React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit and raw HTML templating.
+
+- [Page Routing](/en/core-concepts/astro-pages/#file-based-routing): Nuxt uses `vue-router` for SPA routing, and `vue-meta` for managing `<head>`. In Astro, you will create separate HTML page routes and control your page `<head>` directly, or in a layout component.
+
+- [Content-focus](/en/concepts/why-astro/): Astro was designed primarily for content-focused sites. An existing Nuxt app might be built for high client-side interactivity and may include items that are difficult to replicate in Astro, such as dashboards.
```

Comment:
Heads up, this _might_ have a small hiccup in the fact that [Nuxt has a built-in method for generating pages from Content (`.md`, etc)](https://nuxtjs.org/docs/directory-structure/content/).

While it only seems [to make up about 10% of installs](https://www.npmjs.com/package/@nuxt/content) [compared to the main `nuxt` package](https://www.npmjs.com/package/nuxt), we may not want to assume that these apps coming from Nuxt are not content-focused

Reply:
I've used this wording in a few places (Next.js is one of them) to highlight the "designed for content" Astro vs 
"Next/Nuxt etc. actually can make highly interactive things, so you MIGHT have an app that is hard to replicate."

It's intended to be more a "Next/Nuxt can do this, so you might have this" generous nod to the SPA frameworks while avoiding "Astro can't do" kind of language.  :)  I think I've hedged enough with the wording to make it clear that this only "might" be a problem and you "may" have stuff going on? Let me know if you feel strongly otherwise!